components/browse/BrowseChip.vue

```diff
@@ -1,14 +1,16 @@
 <template>
-  <b-badge
-    v-if="title"
-    :to="localePath({ name: 'entity-type-all', params: { type: type, pathMatch: path }})"
-    pill
-    variant="nocolor"
-    class="mb-3 mr-3 font-weight-normal bg-transparent"
-    data-qa="browse chip"
-  >
-    {{ title }}
-  </b-badge>
+  <li>
+    <b-badge
+      v-if="title"
+      :to="localePath({ name: 'entity-type-all', params: { type: type, pathMatch: path }})"
```

Comment:
While I appreciate this was pre-existing code, the entity route does not belong in this component which should be reusable in different contexts. Let's fix that on this branch.

Instead of accepting `type` and `path` props, accept a `linkTo` prop which could be either an object or string, and have the parent view supply the full entity route object.